### PR TITLE
Update Resume

### PR DESCRIPTION
**Pull request by MajorTreble**
Change display of resume
Update one line item at a time
Added html includes

Task #41

### CODE REVIEW

**Review by SoroorAghamiri — CHANGES_REQUESTED** [review]
I tried running the application, but clicking on Create Profile has errors. See the attached screenshot.
Screenshot 2023-07-19 at 4 15 50 PM

**Comment by MajorTreble** [comment]
> I tried running the application, but clicking on Create Profile has errors. See the attached screenshot. Screenshot 2023-07-19 at 4 15 50 PM

I renamed Users to User but you still have the old table in your db. Please drop all tables.

**Comment by SoroorAghamiri** [comment]
> > I tried running the application, but clicking on Create Profile has errors. See the attached screenshot. Screenshot 2023-07-19 at 4 15 50 PM
> 
> I renamed Users to User but you still have the old table in your db. Please drop all tables.

I couldn't delete it. It says that Users has a foreign key in Resume.

**Comment by MajorTreble** [comment]
> > > I tried running the application, but clicking on Create Profile has errors. See the attached screenshot. Screenshot 2023-07-19 at 4 15 50 PM
> > 
> > 
> > I renamed Users to User but you still have the old table in your db. Please drop all tables.
> 
> I couldn't delete it. It says that Users has a foreign key in Resume.

You need to delete in reverse order, most tables, then resume and then user

**Review by SoroorAghamiri — APPROVED** [review]

**Comment by sepehr9877** [comment]
> I tried running the application, but clicking on Create Profile has errors. See the attached screenshot. Screenshot 2023-07-19 at 4 15 50 PM

so you are getting the same error , go to mysql drop all of your tables and re-run the application it should work . this is mainly because of changing the name of Entity class and not updating the db

File: src/main/resources/templates/profile_employer.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 <html xmlns:th="http://www.thymeleaf.org">
 <head>
-  <title th:text="${employer.firstname} + ' ' + ${employer.lastname} + ' | Whatever Job You Choose'"></title>
+  <section class="content-header" th:replace="include::profile_title"></section>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
 </head>
 <body>
+
```
Comment (sepehr9877) [inline src/main/resources/templates/profile_employer.html:10]: later we can use this th:replace to remove replication in our code specially for navbar

**Review by sepehr9877 — APPROVED** [review]

**Pull request merged**